ACadSharp/IO/DWG/DwgStreamReaders/DwgObjectReader.cs

```diff
@@ -4534,8 +5106,8 @@ private CadTemplate readVisualStyle()
 			visualStyle.Type = this._objectReader.ReadBitLong();
 
 #if TEST
-			var objValues = DwgStreamReaderBase.Explore(_objectReader);
-			var textValues = DwgStreamReaderBase.Explore(_textReader);
+			//var objValues = DwgStreamReaderBase.Explore(_objectReader);
+			//var textValues = DwgStreamReaderBase.Explore(_textReader);
```

Comment:
`readVisualStyle()` is outside the scope of this task, please revert the changes.